src/main/java/com/revature/app/dao/VisualizationDao.java

```diff
@@ -0,0 +1,5 @@
+package com.revature.app.dao;
+
+public class VisualizationDao {
```

Comment:
Add @Repository tag to this class.
Also, change this from a class to an interface and extend the JpaRepository interface like this:
"extends JpaRepository<Visualization, Integer>"